core/src/main/java/com/microsoft/applicationinsights/internal/util/ThreadPoolUtils.java

```diff
@@ -58,6 +59,8 @@ public static void stop(ThreadPoolExecutor threadPool, long timeout, TimeUnit ti
         } catch (InterruptedException ie) {
             threadPool.shutdownNow();
             Thread.currentThread().interrupt();
+            InternalLogger.INSTANCE.error("Interrupted exception occured while closing the thread");
```

Comment:
This is not an error. It can be part of normal operation. I wouldn't log anything here; not even trace.